tests/foreman/api/test_subscription.py

```diff
@@ -181,3 +189,56 @@ def test_positive_delete_manifest_as_another_user(self):
             data={'organization_id': org.id}
         )
         self.assertEquals(0, len(Subscription.list({'organization-id': org.id})))
+
+    @tier2
+    def test_positive_subscription_status_disabled(self):
+        """Verify that Content host Subscription status is set to 'Disabled'
+         for a golden ticket manifest
+
+        :id: d7d7e20a-e386-43d5-9619-da933aa06694
+
+        :expectedresults: subscription status is 'Disabled'
+
+        :BZ: 1789924
+
+        :CaseImportance: Medium
+        """
+        org = entities.Organization().create()
+        with manifests.clone(name='golden_ticket') as manifest:
+            upload_manifest(org.id, manifest.content)
+        rh_repo_id = enable_rhrepo_and_fetchid(
+            basearch='x86_64',
+            org_id=org.id,
+            product=PRDS['rhel'],
+            repo=REPOS['rhst7']['name'],
+            reposet=REPOSET['rhst7'],
+            releasever=None,
+        )
+        rh_repo = entities.Repository(id=rh_repo_id).read()
+        rh_repo.sync()
+        custom_repo = entities.Repository(
+            product=entities.Product(organization=org).create(),
+        ).create()
+        custom_repo.sync()
+        lce = entities.LifecycleEnvironment(organization=org).create()
+        cv = entities.ContentView(
+            organization=org, repository=[rh_repo_id, custom_repo.id],
+        ).create()
+        cv.publish()
+        cvv = cv.read().version[0].read()
+        promote(cvv, lce.id)
+        ak = entities.ActivationKey(
+            content_view=cv, max_hosts=100, organization=org, environment=lce, auto_attach=True
+        ).create()
+        subscription = entities.Subscription(organization=org).search(
+            query={'search': 'name="{}"'.format(DEFAULT_SUBSCRIPTION_NAME)}
+        )[0]
+        ak.add_subscriptions(data={'quantity': 1, 'subscription_id': subscription.id})
+        with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+            vm.install_katello_ca()
+            vm.register_contenthost(org.label, ak.name)
+            assert vm.subscribed
+            host = entities.Host().search(query={'search': 'name={}'.format(vm.hostname)})
+            host_id = host[0].id
```

Comment:
this is just a nitpick and I'm not going to block the pr for this one, but it'd be better to check the `host` list before blindly touching its 1st element, and handle potential errors properly; otherwise, if something goes wrong it might give you `IndexError` without a proper context and the test will be marked as `Error` instead of `Failed`.

Reply:
The value of "host" has only 1 index. I just used it as a way to step into the class list itself.

Reply:
I understand and in a happy path scenario it works just fine. I was referring to a situation where something goes wrong and suddenly the `entities.Host().search()` returns 0 results.
Anyway, it's fine - as I said, just a non-blocking nitpick